setup.examples/apache/mailwatch-apache.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if ( type "apache2" > /dev/null 2>&1 ); then
+  apacheBin="apache2"
+else
+  apacheBin="httpd"
+fi
+if [ -z $("$apacheBin" -v | grep "Apache/2.4") ]
+    sed -i -e "s/ALLGRANTED/Require all granted/" "$DIR/etc/apache2/conf-enabled/mailwatch.conf"
+  else
+    sed -i -e "s/ALLGRANTED/Order allow,deny\n  Allow from all/" "$DIR/etc/apache2/conf-enabled/mailwatch.conf"
+fi
+
+sed -i -e "s/WEBFOLDER/$WebFolder/" "$DIR/etc/apache2/conf-enabled/mailwatch.conf"
```

Comment:
+sed -i -e "s/WEBFOLDER/$WebFolder/" "$DIR/etc/apache2/conf-enabled/mailwatch.conf"
+cp "$DIR/etc/apache2/conf-enabled/mailwatch.conf" /etc/apache2/conf-enabled/mailwatch.conf

For Debian 8 and Apache 2.4

in conf-available

a2enconf mailwatch.conf